emscripten/README.md

```diff
@@ -1,9 +1,11 @@
 building
 --------------------------------------------------------------------------------
-1. compile Uncrustify normally
-2. copy the generated headers into the src directory 
-   (uncrustify_version.h, config.h, token_names.h)
-3. run the build.sh in the emscripten directory to generate libUncrustify.js
+ 1. create a 'build' directory somewhere
+ 2. inside this directory call 'cmake <CMakeLists.txt directory>' and 'make'
+      the CMakeLists.txt file is located in <uncrustify_root_dir>/emscripten/ 
+      dependencies: cmake, python, clang, emscripten, node
+[3.] to test the generated libUncrustify.js call 'make emscripten_test'
```

Comment:
I wanted to highlight that this is optional. I guess a better solution would be to mention this as an extra sentence without numbering and brackets.